locals.tf

```
@@ -0,0 +1,30 @@
+########################################
+# Locals
+########################################
+locals {
+  cloudwatch_log_group_name_prefix = "/ecs/${var.name}-"
+  container_name                   = coalesce(var.container_name, "api-${var.name}")
+  # cannot be longer than 32 chars
+  ecs_exec_iam_role_name_prefix = "${substr(var.name, 0, 22)}-ecs-exec-"
+  ecs_task_iam_role_name_prefix = "${substr(var.name, 0, 22)}-ecs-task-"
+  # cannot be longer than 6 chars
+  lb_target_group_name_prefix = "${substr(var.name, 0, 2)}-tg-"
```

```suggestion
  lb_target_group_name_prefix = "${substr(var.name, 0, 5)}-"
```
Just a thought that maybe we don't need to say tg when it's a tg.. and pick up some extra letters for better context?